docs/content.zh/docs/connectors/table/jdbc.md

```diff
@@ -212,6 +212,14 @@ ON myTopic.key = MyUserTable.id;
       它决定了每个语句是否在事务中自动提交。有些 JDBC 驱动程序，特别是
       <a href="https://jdbc.postgresql.org/documentation/head/query.html#query-with-cursor">Postgres</a>，可能需要将此设置为 false 以便流化结果。</td>
     </tr>
+    <tr>
+      <td><h5>scan.parallelism</h5></td>
```

Comment:
could you add a link to some documentation for this - as there is for autocommit please.